src/Store/slices/testStringSlice.tsx

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const greetingSlice  = createSlice({
+  name: "greeting",
+  initialState: "Welcome to my project!",
```

Comment:
create an interface to type your state, but if it's only to test you can do it later on